src/lib/cli.ts

```diff
@@ -61,6 +69,22 @@ const setUpUserFeedback = (sonarInstance: Sonar, spinner: IORA) => {
     });
 };
 
+type Update = { // eslint-disable-line no-unused-vars
+    latest: string,
+    current: string,
+    type: string,
+    name: string
+};
+
+const notifyNewVersion = (update: Update) => {
+    const changelogUrl: string = `https://sonarwhal.com/about/changelog.html`;
+    // No indentation due to the use of `\` to avoid new line.
+    // https://stackoverflow.com/a/35428171
+    const message: string = `Update available ${chalk.dim(update.current)}${chalk.reset(' → ')}${chalk.green(update.latest)}\nRun ${chalk.cyan(`npm i -g ${update.name}`)} to update\nSee ${chalk.red(changelogUrl)} for update details`;
```

Comment:
@alrra If we have a label in the code that indicates whether or not sonar is installed as a dependency, I can generate different messages based on different scenarios. Does such "label" exist? Otherwise I can include both `npm i -g` and `npm i -D` options in the message and ask the user to pick according to their own use.